docs/source/install.md

```diff
@@ -34,9 +31,26 @@ This will pull any necessary docker containers, and start the system in the back
 The default system includes a small amount of test data. For a test system with considerably more data you can run
 
 ```bash
-bash <(curl -s https://raw.githubusercontent.com/Flowminder/FlowKit/master/quick_start.sh) synth
+bash <(curl -s https://raw.githubusercontent.com/Flowminder/FlowKit/master/quick_start.sh) larger_data
+```  
+
+!!! warning
+    The larger data container will take considerably longer to start up, as it generates data when first run.
+
+The [worked examples](worked_examples) are also available as part of the demo system. To install these run 
+
+```bash
+bash <(curl -s https://raw.githubusercontent.com/Flowminder/FlowKit/master/quick_start.sh) examples
+```  
+
+for the examples with a small dataset, or 
+
+```bash
+bash <(curl -s https://raw.githubusercontent.com/Flowminder/FlowKit/master/quick_start.sh) examples larger_data
```

Comment:
Would be useful to also give instructions here for stopping the system cleanly (presumably `bash ... stop` is sufficient).